packages/@react-aria/select/docs/useSelect.mdx

```diff
@@ -520,3 +520,9 @@ content that is passed into the select.
 In right-to-left languages, the select should be mirrored. The arrow should be on the left,
 and the selected value should be on the right. In addition, the content of list options should
 flip. Ensure that your CSS accounts for this.
+
+### False positive on `<HiddenSelect />` in accessibility tools
+
+When using &lt;<TypeLink links={docs.links} type={docs.exports.HiddenSelect} />&gt; some accessibility tools such as AXE or pa11y might give a false positive.
+The data attribute `data-rsp-a11y-ignore` is included on the hidden select element and can be used to ignore the false positives in these tools.
```

Comment:
For the team, open to opinions on where this information should go. Perhaps we should add a new section in this docs page for accessibility gotchas (or simply accessiblity?)

Reply:
I'm fine with where it is, but not against a dedicated accessibility section either.

Reply:
+1 to Reid's comment. I think this is fine where it is but a new section for accessibility gotchas would be nice

Reply:
Alright, I'll approve this PR and make a followup one moving it to a separate accessibility section.

Reply:
It definitely shouldn't be under "Internationalization" IMO, seems unrelated to that. Also we will need to cover this on all of the other select pages: RAC Select and RSP Picker. If it is going to be a general way for us to indicate false positives, we may also want to document it on a more general page such as the "Accessibility" one.

Reply:
good point about covering this on the other Select pages. By the "Accessibility" page I assume you are referencing [this](https://react-spectrum.adobe.com/react-aria/accessibility.html#accessibility)? If so, I think that is a good idea.

@abitbetterthanyesterday IMO this is creeping in scope a bit, would you like to take this on? I'd be perfectly happy making followup PR for this as well instead.